### PR TITLE
Modify source build using latest tag, docker build using quay.io images

### PR DESCRIPTION
**Pull request by xiuwang**
@openshift/devexp-qe please help review, thanks

### CODE REVIEW

File: features/build/buildconfig.feature [hunk 1]
```diff
@@ -26,37 +26,40 @@ Feature: buildconfig.feature
   Scenario: Rebuild image when the underlying image changed for Docker build
     Given I have a project
     When I run the :new_build client command with:
-      | app_repo | https://github.com/openshift/ruby-hello-world.git |
+      | D    | FROM quay.io/openshifttest/centos@sha256:285bc3161133ec01d8ca8680cd746eecbfdbc1faa6313bd863151c4b26d7e5a5\nRUN echo "hello" |
+      | to   | centos  |
+      | name | mybuild |
+      #| app_repo | https://github.com/openshift/ruby-hello-world.git |
```
Comment (liangxia) [inline features/build/buildconfig.feature:32]: If the line is not longer valid, consider remove it rather than comment out.

Comment (xiuwang) [reply features/build/buildconfig.feature]: @liangxia thanks to point it out, remove this comment line. Please help review again.

**Review by pruan-rht — COMMENTED** [review]
/lgtm

**Pull request merged**